operator/charts/helm/rabbitmq/templates/pod-disruption-budget.yaml

```diff
@@ -0,0 +1,19 @@
+{{ if not .Values.externalRabbitmq.enabled }}
+{{- if .Values.rabbitmq.disruptionBudget.enabled }}
+{{- if .Capabilities.APIVersions.Has "policy/v1/PodDisruptionBudget" }}
+apiVersion: policy/v1
+{{- else }}
+apiVersion: policy/v1beta1
+{{- end }}
+kind: PodDisruptionBudget
+metadata:
+  name: rabbitmq-pdb
+  labels:
+    {{- include "rabbitmq.defaultLabels" . | nindent 4 }}
+spec:
+  maxUnavailable: {{ template "rabbitmq.pdb.maxUnavailable" . }}
+  selector:
+    matchLabels:
+      component: rabbitmq
```

Comment:
I didn’t find any component labels. As far as I can see, the StatefulSet is generated by the rabbit-operator, so there is a static value:
sts_labels["app"] = "rmqlocal"
https://github.com/Netcracker/qubership-rabbitmq/blob/bd67b8c322a231ee5b9ca97726463de7ee65a7c4/operator/src/handler.py#L772

Please check and fix.